### PR TITLE
CFDE KF RNASeq Edits

### PR DESCRIPTION
**Pull request by HigginsD3b**
## PR Checklist

- [X] Release label added
- [X] PR category label added
- [X] PR mergeable
- [X] Clean build logs
- [X] Linked related issues
- [X] Colorblindness test https://www.toptal.com/designers/colorfilter/

## PR Description
Briefly describe the changes this PR will add/address: Suggested changes to the Kids First RNASeq training documentation from the Kids First DRC.

## Review format
Add how you would like to receive the review (direct editing, as comments, notes etc): Direct editing is welcome. The changes are mostly small adjustments intended to improve clarity or grammar. Overall this is a great set of documents, thank you for putting them together!

## Timeline
Add expected timeline to finish review and merge adding note for time critical updates: As you see fit within your schedules.

### CODE REVIEW

File: docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_2.md [hunk 1]
```diff
@@ -31,18 +32,8 @@ The generated raw reads are quality checked and aligned against either a referen
...
 
-* is a broad group of tumors
-* often arises from lining of the ventricles in the brain
-* can also occur in the central canal in the spinal cord
-* anatomical distribution impacts prognosis
+**Ependymoma** is a broad group of tumors that often arises from lining of the ventricles in the brain. Ependymomas can also occur in the central canal in the spinal cord, with the anotomical distribution impacting prognosis.
 
-We will use the [Kids First Data Portal (KF Portal)](https://kidsfirstdrc.org) to build a virtual cohort containing the two pediatric cancers and select pre-processed transcript abundance files. We will then proceed with analysis on [Cavatica](https://cavatica.sbgenomics.com), the integrated cloud based platform.
+We will use the [Kids First Data Portal (KF Portal)](https://kidsfirstdrc.org) to build a virtual cohort of tumor samples of the two pediatric cancers. We will us the KF Portal to select pre-processed transcript abundance files, which we will then analyze on [Cavatica](https://cavatica.sbgenomics.com), the KF Portal's integrated cloud based platform.
```
Comment (marisalim) [inline docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_2.md]: typo fix - "We will use the KF Portal to select pre-processed..."

File: docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_3.md [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
 
-!!! info "KFDRC RNAseq workflow"
+!!! info "KFDRC RNAseq Workflow"
 
-    [Kids First RNAseq pipeline](https://github.com/kids-first/kf-rnaseq-workflow) uses multiple tools/packages for expression detection and fusion calls. The workflow requires raw FASTQ files (controlled access) as input and generates multiple outputs including the Kallisto transcript quantification files. All the output files of this pipeline are available on the portal as open access data. In addition to the restricted data access issue, it is computationally taxing to run this workflow on multiple files.
+    The [Kids First RNAseq Workflow](https://github.com/kids-first/kf-rnaseq-workflow) uses multiple tools/packages for expression detection and fusion calls. The workflow requires raw FASTQ files (controlled access) as input and generates multiple outputs including the Kallisto transcript quantification files. All the output files of this pipeline are available on the KF Portal as open access data. Due to access restrictions and computational intensiveness, this tutorial will not cover this workflow, but users with their own RNAseq data could consider starting here.
```
Comment (marisalim) [inline docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_3.md]: suggestion: "Due to access restrictions and computational intensity, this tutorial will not cover the Kallisto workflow, but users with their own RNAseq data may consider starting with it."

File: docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_3.md [hunk 3]
```diff
@@ -1,39 +1,39 @@
...
+* Login to the [KF Portal](https://kidsfirstdrc.org/).
+* Select the <span class="highlight_txt">File Repository</span> tab.
 * Select the <span class="highlight_txt">Browse All</span> option for the Filter.
 
 ![File Repository](../rna-seq-images/1_KFDRC.png "File Repository")
 
 !!! note "Data Summary"
 
-    At the time of the tutorial (Jan 2021), the portal contained a total of 88,728 files. Since new datasets are constantly uploaded to the KF portal, the query numbers may change when run in the future.
+    At the time this tutorial was written (January 2021), the portal contained a total of 88,728 files. Because new datasets are constantly uploaded to the KF Portal, exact numbers within your query may change slighly when run in the future.
```
Comment (marisalim) [inline docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_3.md]: typo fix - "query may change slightly"

File: docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_4.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ title: Cavatica - View, Filter, Tag and Download
 Cavatica - View, Filter, Tag and Download
 =======================
 
-To view the project folder on Cavatica, you can click the link from the pop box in KF portal after successful copy of files which will open the Cavatica login page.
+To view the project folder on Cavatica, you can click the link from the pop box in KF Portal after successfully copying files. This will will open the Cavatica login page.
```
Comment (marisalim) [inline docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_4.md]: typo fix - "click the link from the pop-up box"

File: docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_4.md [hunk 5]
```diff
@@ -87,7 +86,7 @@ To enable quick access to the filtered data without having to re-run all the met
 
 !!! tip "Tag Names"
 
-    While you can use any tag name you see fit, use **DGE-FILTER-DATA** as used in this lesson to match your screen with the lesson screenshots.
+    While you can use any tag name you see fit, we recommend using **DGE-FILTER-DATA**, used in this lesson to match your screen with the lesson screenshots.
```
Comment (marisalim) [inline docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_4.md]: suggestion: "You can use any tag name. In this lesson and in the screenshots, we use **DGE-FILTER-DATA**."

**Comment by marisalim** [comment]
Thanks for the edits @HigginsD3b! I made some suggestions above that would be great to fix before we merge the PR.

**Review by HigginsD3b — COMMENTED** [review]
Thanks for the feedback Marisa - I made the changes you suggested!

**Review by marisalim — APPROVED** [review]
looks good to merge - thanks for these edits @HigginsD3b!

**Pull request merged**